### PR TITLE
Major version change kubernetes

### PR DESCRIPTION
**Pull request by Anumita**
Drop down to select azure kubernetes cluster
Remove json input from registry inside create secret section. Just have registry name.
Remove output variable section from kubernetes task.
Not download kubernets cli with every task execution instead use the kubernetes utility in Common.

### CODE REVIEW

File: Tasks/KubernetesV1/Strings/resources.resjson/en-US/resources.resjson [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  "loc.input.help.azureSubscriptionEndpoint": "Select the Azure Resource Manager subscription, which contains Azure Container Registry.Note: To configure new service connection, select the Azure subscription from the list and click 'Authorize'. If your subscription is not listed or if you want to use an existing Service Principal, you can setup an Azure service connection using 'Add' or 'Manage' button.",
+  "loc.input.label.azureResourceGroup": "Resource group",
+  "loc.input.help.azureResourceGroup": "Select an Azure resource group.",
+  "loc.input.label.kubernetesCluster": "Kubernetes cluster",
+  "loc.input.help.kubernetesCluster": "Select an Azure managed cluster.",
+  "loc.input.label.namespace": "Namespace",
+  "loc.input.help.namespace": "Set the namespace for the kubectl command by using the –namespace flag. If the namespace is not provided, the commands will run in the default namespace.",
+  "loc.input.label.command": "Command",
+  "loc.input.help.command": "Select or specify a kubectl command to run.",
+  "loc.input.label.useConfigurationFile": "Use Configuration files",
```
Comment (vithati) [inline Tasks/KubernetesV1/Strings/resources.resjson/en-US/resources.resjson]: Use configuration files

File: Tasks/KubernetesV1/Strings/resources.resjson/en-US/resources.resjson [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  "loc.input.help.useConfigurationFile": "Use Kubernetes configuration file with the kubectl command. Filename, directory, or URL to Kubernetes configuration files can also be provided.",
+  "loc.input.label.configuration": "Configuration file",
+  "loc.input.help.configuration": "Filename, directory, or URL to kubernetes configuration files that will be used with the commands.",
+  "loc.input.label.arguments": "Arguments",
+  "loc.input.help.arguments": "Arguments to the specified kubectl command.",
+  "loc.input.label.secretType": "Type of secret",
+  "loc.input.help.secretType": "Create/update a generic or dockerimagepullsecret.",
+  "loc.input.label.secretArguments": "Arguments",
+  "loc.input.help.secretArguments": "Specify keys and literal values to insert in secret.For example, --from-literal=key1=value1 --from-literal=key2=\"top secret\".",
+  "loc.input.label.containerRegistryType": "Container Registry type",
```
Comment (vithati) [inline Tasks/KubernetesV1/Strings/resources.resjson/en-US/resources.resjson]: Container registry type

File: Tasks/KubernetesV1/Strings/resources.resjson/en-US/resources.resjson [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  "loc.input.help.containerRegistryType": "Select a Container registry type. The task can use Azure Subscription details to work with an Azure Container registry. Other standard Container registries are also supported.",
+  "loc.input.label.dockerRegistryEndpoint": "Docker Registry connection",
```
Comment (vithati) [inline Tasks/KubernetesV1/Strings/resources.resjson/en-US/resources.resjson]: Docker registry connection

File: Tasks/KubernetesV1/Strings/resources.resjson/en-US/resources.resjson [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+  "loc.input.help.dockerRegistryEndpoint": "Select a Docker registry connection. Required for commands that need to authenticate with a registry.",
+  "loc.input.label.azureSubscriptionEndpointForSecrets": "Azure subscription",
+  "loc.input.help.azureSubscriptionEndpointForSecrets": "Select the Azure Resource Manager subscription, which contains Azure Container Registry. Note: To configure new service connection, select the Azure subscription from the list and click 'Authorize'. If your subscription is not listed or if you want to use an existing Service Principal, you can setup an Azure service connection using 'Add' or 'Manage' button.",
+  "loc.input.label.azureContainerRegistry": "Azure Container Registry",
```
Comment (vithati) [inline Tasks/KubernetesV1/Strings/resources.resjson/en-US/resources.resjson]: Azure container registry

File: Tasks/KubernetesV1/Strings/resources.resjson/en-US/resources.resjson [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+  "loc.input.label.configMapFile": "ConfigMap file",
+  "loc.input.help.configMapFile": "Specify a file or directory that contains the configMaps",
+  "loc.input.label.configMapArguments": "Arguments",
+  "loc.input.help.configMapArguments": "Specify keys and literal values to insert in configMap.For example, --from-literal=key1=value1 --from-literal=key2=\"top secret\".",
+  "loc.input.label.versionOrLocation": "Kubectl",
+  "loc.input.label.versionSpec": "Version spec",
+  "loc.input.help.versionSpec": "Version Spec of version to get.  Examples: 1.7.0, 1.x.0, 4.x.0, 6.10.0, >=6.10.0",
+  "loc.input.label.checkLatest": "Check for latest version",
+  "loc.input.help.checkLatest": "Always checks online for the latest available version (stable.txt) that satisfies the version spec. This is typically false unless you have a specific scenario to always get latest. This will cause it to incur download costs when potentially not necessary, especially with the hosted build pool.",
+  "loc.input.label.specifyLocation": "Path to Kubectl",
```
Comment (vithati) [inline Tasks/KubernetesV1/Strings/resources.resjson/en-US/resources.resjson]: Path to kubectl

File: Tasks/KubernetesV1/Tests/package-lock.json [hunk 6]
```diff
@@ -0,0 +1,107 @@
+{
```
Comment (vithati) [inline Tasks/KubernetesV1/Tests/package-lock.json]: Remove this file.

**Comment by stephenmichaelf** [comment]
@Anumita FYI we are generally trying to get away from Typings and replace it with @ types.

**Pull request merged**